integration-tests/Dockerfile

```diff
@@ -1,2 +1,7 @@
-FROM python:2-onbuild
-CMD [ "python", "./osmo-smsc-integrationtests.py", "-H", "osmo-smsc-om"]
+FROM python:2-alpine
```

Comment:
Main reason to switch is, that the alpine image uses only 80mb of space, where the onbuild uses 400 or more. So it saves us some downloadtime. The other diffrence is that the on-build has already the pip install stanca.